tests/llmcompressor/transformers/kv_cache/test_kv_cache.py

```diff
@@ -0,0 +1,330 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+import torch
+from accelerate import init_empty_weights
+from compressed_tensors.quantization.lifecycle import KVCacheScaleType
+from compressed_tensors.quantization.utils.helpers import iter_named_quantizable_modules
+from datasets import load_dataset
+from parameterized import parameterized_class
+from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
+
+from llmcompressor.core import reset_session
+from llmcompressor.transformers import oneshot
+from tests.testing_utils import parse_params
+
+NUM_CALIBRATION_SAMPLES = 16
+MAX_SEQUENCE_LENGTH = 512
+DATASET_ID = "HuggingFaceH4/ultrachat_200k"
+DATASET_SPLIT = "train_sft"
+
+KV_CACHE_CONFIGS_DIRECTORY = (
+    "tests/llmcompressor/transformers/kv_cache/configs/kv_cache"
+)
+KV_CACHE_GPTQ_CONFIGS_DIRECTORY = (
+    "tests/llmcompressor/transformers/kv_cache/configs/gptq_kv_cache"
+)
+
+
+@parameterized_class(parse_params(KV_CACHE_CONFIGS_DIRECTORY))
```

Comment:
Have we confirmed this works with the current e2e set-up?
Does the bash command need to be updated?